src/services/item/types.ts

```diff
@@ -17,6 +17,17 @@ export enum Ordering {
   DESC = 'DESC',
 }
 
+export function orderingToUpperCase(ordering: Ordering): Ordering.ASC | Ordering.DESC {
```

Comment:
not sure to understand the purpose of this

Reply:
Because TypeORM doesn't accept 'asc' or 'desc' as ordering, and our API accept this kind of input (but not `AsC` or `deSc`) we need to normalize it with toUpperCase().
I don't like having a typecast in our repository, so I kinda moved it here.. I've not found any better solution, I'm open to suggestions

Reply:
I think it's a good solution in this case :).